playbooks/roles/edx_django_service/templates/edx/app/nginx/sites-available/discovery.j2

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
@amirtds This shouldn't need to happen. Maintaining this large template could be an issue.

A better solution would be the following, add a task to do the following:

 - Make `ssl_certificate /etc/ssl/certs/{{ NGINX_SSL_CERTIFICATE|basename }};` links to `{{ NGINX_SSL_CERTIFICATE_PATH }}`
 - Same goes for `NGINX_SSL_KEY`

Reply:
I agree with @OmarIthawi this is really dirty. We shouldn't need to customize this script just for the discovery role in order to get SSL to work.